src/PockyBot.NET/Services/UsernameUpdater.cs

```diff
@@ -38,5 +43,18 @@ public async Task<List<PockyUser>> UpdateUsernames(List<PockyUser> users)
             await Task.WhenAll(dbUpdates).ConfigureAwait(false);
             return updatedUsers;
         }
+
+        private Task<Person> GetUser(string userId, string username)
+        {
+            try
+            {
+                return _personHandler.GetPersonAsync(userId);
+            }
+            catch (HttpRequestException ex)
+            {
+                _logger.LogError($"Error retrieving details for user {userId}");
+                return Task.FromResult(new Person{UserId = userId, Username = username});
```

Comment:
this isn't going to do anything weird like try to send a PM to a non-existent user, is it?

Reply:
hmmm... worth investigating, good point